run_testlist.py

```diff
@@ -32,972 +33,606 @@
+            'ppc': 32
+        },
+        'enabled': False
+    },
+    'ec_ior_full_stripe_EC_16P2GX': {
+        'scale': [
+            # (num_servers, num_clients, timeout_minutes)
+            (18, 36, 5)
+        ],
+        'ec_cell_size': [
+            (65536),
+            (1048576)
+        ],
+        'oclass': 'EC_16P2GX',
+        'cont_rf': '2',
+        'env_vars': {
+            'chunk_size': '33554432',
+            'pool_size': '85G',
+            'segments': '1',
+            'xfer_size': '16M',
+            'block_size': '1G',
+            'fpp': '-F',
+            'sw_time': '60',
+            'iterations': '2',
+            'ppc': 32
+        },
+        'enabled': False
+    }
+}
+
+
+ior_single_replica_testdict = {
+    'ior_easy_S2': {
+        'scale': [
+            # (num_servers, num_clients, timeout_minutes)
+            (4, 8, 5)
+        ],
+        'oclass': 'S2',
+        'env_vars': {
+            'chunk_size': '1048576',
+            'pool_size': '85G',
+            'segments': '1',
+            'xfer_size': '2M',
+            'block_size': '1G',
+            'fpp': '-F',
+            'sw_time': '60',
+            'iterations': '2',
+            'ppc': 32
+        },
+        'enabled': False
+    },
+    'ior_easy_S4': {
+        'scale': [
+            # (num_servers, num_clients, timeout_minutes)
+            (6, 12, 5)
+        ],
+        'oclass': 'S4',
+        'env_vars': {
+            'chunk_size': '1048576',
+            'pool_size': '85G',
+            'segments': '1',
+            'xfer_size': '4M',
+            'block_size': '1G',
+            'fpp': '-F',
+            'sw_time': '60',
+            'iterations': '2',
+            'ppc': 32
+        },
+        'enabled': False
+    },
+    'ior_easy_S8': {
+        'scale': [
+            # (num_servers, num_clients, timeout_minutes)
+            (40, 20, 5)
+        ],
+        'oclass': 'S8',
+        'env_vars': {
+            'chunk_size': '1048576',
+            'pool_size': '85G',
+            'segments': '1',
+            'xfer_size': '8M',
+            'block_size': '1G',
+            'fpp': '-F',
+            'sw_time': '60',
+            'iterations': '2',
+            'ppc': 32
+        },
+        'enabled': False
+    },
+    'ior_easy_S16': {
+        'scale': [
+            # (num_servers, num_clients, timeout_minutes)
+            (18, 36, 10)
+        ],
+        'oclass': 'S16',
+        'env_vars': {
+            'chunk_size': '1048576',
+            'pool_size': '85G',
+            'segments': '1',
+            'xfer_size': '16M',
+            'block_size': '1G',
+            'fpp': '-F',
+            'sw_time': '60',
+            'iterations': '2',
+            'ppc': 32
+        },
+        'enabled': False
+    },
+}
+
+
+ior_testdict = {
+    'ior_easy': {
+        'oclass': [
+            #'SX',
+            #'RP_2GX',
+            #'RP_3GX',
+            #'EC_2P1GX',
+            #'EC_4P1GX'
```

Comment:
Disabling the standard tests by default since there are so many variants. Don't want to accidentally fire off 100+ tests. Also, this doesn't yet include all the EC tests I was working on. That part will need some further refactoring